forest/willow/log_stats.py

```diff
@@ -140,6 +140,69 @@ def text_analysis(
     )
 
 
+def text_and_call_analysis(
+    df_call: pd.DataFrame, df_text: pd.DataFrame, stamp: int, step_size: int
+) -> tuple:
+    """Calculate the summary statistics for the call data
+    in the given time interval.
+
+    Args:
+        df_call: pd.DataFrame
+            dataframe of the call data
+        df_text: pd.DataFrame
+            dataframe of the text data
+        stamp: int
+            starting timestamp of the study
+        step_size: int
+            ending timestamp of the study
```

Comment:
Different description?

Reply:
Ah, thanks for catching that. That should be updated